README.md

```diff
@@ -37,10 +37,48 @@ To only start the client:
 $ yarn start
 ```
 
+### Starting dockerized development environment
+
+1. Check if Docker and docker CLI installed, port `3000` and `9000` is free, not occupied by running server.
+
+2. Make sure you have env variables in `.env.development.local`, otherwise extend it from example by:
+    ```
+    $ cp .env.example .env.development.local
+    ```
+3. Start building docker image and start container:
+    ```
+    $ docker-compose up
+    ```
+4. Open `localhost:3000` on browser.
+
 ## Testing
 
 End-to-end testing is created with Cypress.io framework. To run tests:
 
 ```
 $ yarn test
 ```
+
+## Useful docker command
+- To rebuild the docker images:
+    ```
+    $ docker-compose up --force-recreate --build
+    ```
+- To enter inside docker container environment:
+    ```
+    $ docker-compose exec web sh
```

Comment:
` docker exec berth-frontend sh`?

Reply:
Should i name the image @igordavydsson ?

Reply:
@igordavydsson : Btw the script is correct, exec [SERVICE_NAME] not [CONTAINER_NAME]

Reply:
ah, I was thinking about `docker exec` not `docker-compose exec` 😄 my bad.